examples/file-context-example.yml

```
@@ -0,0 +1,60 @@
+
+# Define system prompt with file-based context
+systemPrompt:
+  contributors:
+    # Main agent instructions
+    - id: base-prompt
+      type: static
+      priority: 0
+      content: |
+        You are a helpful coding assistant with knowledge of the current project.
+        You have access to project documentation and context files.
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)